blockchain/chain_sync/src/sync.rs

```diff
@@ -274,7 +274,7 @@ where
         };
         info!(
             "Received block over GossipSub: {} from {}",
-            block.header.epoch(),
+            block.header.cid(),
```

Comment:
Could you also print out the height? It makes it easier to visualize whats going on.